src/plugins/ssh/index.ts

```diff
@@ -63,6 +63,7 @@ const DEFAULT_VALIDATION_WINDOW_MS = 5e3;
  * Note that each attempt consumes ~ 1 s.
  */
 const MAX_SSH_RETRIES = 30;
+const GCP_MAX_SSH_RETRIES = 120; // GCP requires more time to propagate access
```

Comment:
How long does the SSH attempt take with GCP? I presume the jsdoc comment assumes AWS.

If the consumed time per attempt is significantly shorter, should we use a sleep instead of retrying more?

Reply:
That depends on the type of issue with the access.
iirc 
> 3. The user cannot act as the service account of the compute instance
takes the longest on the order of several seconds, while most others are really quick, under 1s.

So the nice solution would be to remember the start time and sleep based on elapsed time. I will give that a stab.

Reply:
In the interest of time I'm going to punt on this in the current PR.